.gitmodules

```diff
@@ -7,9 +7,6 @@
 [submodule "pikvm-rest-api"]
 	path = pikvm-rest-api
 	url = git@github.com:3mdeb/pikvm-rest-api.git
-[submodule "lib"]
-	path = lib
-	url = git@github.com:3mdeb/testing-stands.git
```

Comment:
why are we removing this submodule? is it no longer needed?

Reply:
Not used. There are a few more things to remove from this repo I have missed removing when moving from githlab to github.

Reply:
Thanks, I just didn't see it explained in the commit message.